client/containers/DashboardSettings/CommunitySettings/ExportCommunityDataButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button } from '@blueprintjs/core';
+import { apiFetch } from 'client/utils/apiFetch';
+import { pingTask } from 'client/utils/pingTask';
+import React, { useState } from 'react';
+
+export const ExportCommunityDataButton = ({ disabled }: { disabled?: boolean }) => {
+	const [isExporting, setIsExporting] = useState(false);
+	const [exportUrl, setExportUrl] = useState<string | null>(null);
+
+	return (
+		<div>
+			{exportUrl && (
+				<a href={exportUrl} target="_blank" rel="noopener noreferrer">
+					Download
+				</a>
+			)}
+			<Button
+				disabled={isExporting || disabled}
+				onClick={() => {
+					setIsExporting(true);
+					apiFetch('/api/communities/archive', {
+						method: 'POST',
+						body: JSON.stringify({ dontWait: true }),
```

Comment:
immediately returning a URL here is kind of an awkward experience, since there's no way for a user to know when the link will work (you get an s3 access denied error if you click before the download finishes). since we're already using the `pingTask` helper here, couldn't we keep the button disabled until the download completes?